pkgs/development/python-modules/av/default.nix

```diff
@@ -1,33 +1,141 @@
 { lib
 , buildPythonPackage
-, fetchPypi
-, isPy27
-, numpy
-, ffmpeg
+, fetchFromGitHub
+, pythonOlder
+
+# build
+, cython
 , pkg-config
+
+# runtime
+, ffmpeg
+
+# tests
+, numpy
+, pillow
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "av";
-  version = "8.1.0";
-  disabled = isPy27; # setup.py no longer compatible
+  version = "9.1.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0402169bc27e38e0f44e0e0e1854cf488337e86206b6d25d6dae2bfd7a1a0230";
+  src = fetchFromGitHub {
+    owner = "mikeboers";
+    repo = "PyAV";
+    rev = "v${version}";
+    hash = "sha256-/6C5GE9zANPy0xaptu/+pIupOLDra/R7TJ41YLGszUs=";
   };
 
-  checkInputs = [ numpy ];
+  nativeBuildInputs = [
+    cython
+    pkg-config
+  ];
+
+  buildInputs = [
+    ffmpeg
```

Comment:
We should patch in the path to ffmpeg instead. Otherwise any reverse dependency will also have to add ffmpeg to its `buildInputs`.

Reply:
Ah yeah, thanks!

Reply:
Oh true, never mind.